blog.md

```diff
@@ -1,26 +1,78 @@
 ## About
 
-This blog post describes (and is hosted on) a demonstration of a CI/CD pipeline where each Pull Request creates an ephemeral environment that could be reviewed as part of the github review process as well as automatically deploying and upgrading a staging and a production environment.
+Docker and its nascent ecosystem are interesting, but they aren't solving problems -  just moving them around. For me the real problems get solved when we change the discussion from Infrastructure (as a Service) to Platform (as a Service) and the _unit_ delivered by the Platform being a running application rather than a Virtual Machine.
 
-I chose to demonstrate this using _only_ opensource products. This means I am running everything from the _IAAS_ layer up. You could quite easily skip OpenStack and run this in AWS or Digital Ocean.
+Even Kubernetes to me still feels a lot like IaaS, just with containers being the unit of Infrastructure rather than VMs. Of course there are already Platforms out there that focus the user experience towards the application rather than the infrastructure with Cloud Foundry and Heroku being major players in the space.
```

Comment:
"Of course" is right up there with "obviously" in the list of terms to look out for. They imply that the reader already knows this information, which means either that you didn't need to write it, or that the reader is dumb for not knowing it already. Either way that's bad, heh. The solution here is simple enough: replace "Of course" with "To be fair,".

Consider:
« Even Kubernates still feels a lot like IaaS - just with containers instead of virtual machines. To be fair, there are already some platforms out there that shift the user experience towards the application (Cloud Foundry and Heroku come to mind), but [_follow up here_]. »

* We can trim the first sentence down because it follows logically from the thesis presented just above.
* If you mention that there are already solutions, then you also need to mention why this article needs to exist. This helps to establish relevance (and credibility). It's your call entirely on how to tackle this; maybe those examples aren't good enough, or maybe they're great for some use-cases but you've got something else in mind...